x/observer/types/keys.go

```diff
@@ -17,6 +17,8 @@ const (
 
 	// MemStoreKey defines the in-memory store key
 	MemStoreKey = "mem_observer"
+
+	GroupId1Address = "zeta1afk9zr2hn2jsac63h4hm60vl9z3e5u69gndzf7c99cqge3vzwjzsxn0x73"
```

Comment:
what is this address?

Reply:
A constant used for the first group ID 1, we were using it in multiple places, just replacing it with a constant